keyboards/kudox/rev1/rev1.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "../kudox.h"
```

Comment:
The keyboard level is added as part of the PATH, so you don't need to specify the directory one level up here.

```suggestion
#include "kudox.h"
```